core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -24,12 +24,16 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
```

Comment:
Please delete the `dp` import to add the `DesignSizes` file please.

Reply:
In DesignSizes file the're not exist 15, 5, 45 dp Sizes, for that motive i put sizes in hard code, what can i do @melissaquezada94 ?

Reply:
Could you please ask to irving if you could add them in the DesignSizes file?